adyen.php

```diff
@@ -1061,4 +1022,105 @@ private function isSimplePaymentMethod($paymentMethod)
                 )
             );
     }
+
+    /**
+     * @param $params
+     *
+     * @throws \PrestaShop\PrestaShop\Adapter\CoreException
+     */
+    public function hookActionFrontControllerSetMedia($params)
+    {
+        $controller = $this->context->controller;
+        if ($controller->php_self == 'order') {
+            $this->registerAdyenJavascript($controller);
+        }
+    }
+
+    /**
+     * @param FrontController $controller
+     *
+     * @throws \PrestaShop\PrestaShop\Adapter\CoreException
+     */
+    private function registerAdyenJavascript($controller)
+    {
+        /** @var \Adyen\PrestaShop\service\adapter\classes\Controller $controllerAdapter */
+        $controllerAdapter = $this->getService('Adyen\PrestaShop\service\adapter\classes\Controller');
+        $controllerAdapter->setController($controller);
+        if ($this->helper_data->isDemoMode()) {
+            $controllerAdapter->registerJavascript(
+                'adyen-component', // Unique ID
+                \Adyen\PrestaShop\service\Configuration::CHECKOUT_COMPONENT_JS_TEST, // JS path
+                array('server' => 'remote', 'position' => 'bottom', 'priority' => 150) // Arguments
+            );
+            $controllerAdapter->registerJavascript(
+                'adyen-threeDS2Utils', // Unique ID
+                $this->_path . 'views/js/threeds2-js-utils.js', // JS path
+                array('position' => 'bottom', 'priority' => 160) // Arguments
+            );
+            $controllerAdapter->registerStylesheet(
+                'adyen-stylecheckout', // Unique ID
```

Comment:
would use the same uniq id:
IF the css and js has these uniq ids in a different pool
```suggestion
                'adyen-checkout-component', // Unique ID
```